package.json

```diff
@@ -3,23 +3,33 @@
   "version": "1.0.0",
   "license": "MIT",
   "scripts": {
+    "dev:server": "ts-node ./src/index.ts",
+    "dev:worker": "ts-node ./src/index-worker.ts",
+    "dev": "concurrently yarn:dev:*",
     "build": "tsc",
-    "run:server": "node dist/index.js",
-    "run:worker": "node dist/index-worker.js"
+    "start:server": "node ./dist/index.js",
+    "start:worker": "node ./dist/index-worker.js",
+    "start": "concurrently yarn:start:*",
+    "migration:generate": "ts-node migration generate",
+    "migration:run": "ts-node migration run",
+    "migration:revert": "ts-node migration revert"
   },
   "dependencies": {
-    "@vendure/admin-ui-plugin": "^1.6.1",
-    "@vendure/asset-server-plugin": "^1.6.1",
-    "@vendure/core": "^1.6.1",
-    "@vendure/email-plugin": "^1.6.1",
-    "pg": "^8.4.1",
-    "typescript": "4.3.5"
+    "@vendure/admin-ui-plugin": "2.1.9",
+    "@vendure/asset-server-plugin": "2.1.9",
+    "@vendure/core": "2.1.9",
+    "@vendure/email-plugin": "2.1.9",
+    "dotenv": "16.4.5",
+    "pg": "8.11.5",
+    "typescript": "4.9.5"
   },
   "devDependencies": {
     "@types/fs-extra": "^9.0.1",
     "@types/node": "^14.6.0",
-    "@vendure/create": "^1.6.1",
-    "@vendure/testing": "^1.6.1",
-    "fs-extra": "^9.0.1"
+    "@vendure/create": "^2.1.9",
+    "@vendure/testing": "^2.1.9",
+    "concurrently": "8.2.2",
+    "fs-extra": "^9.0.1",
+    "ts-node": "10.9.2"
```

Comment:
I did a vendure-create with postgres - and then transferred the dependencies. Unsure if some of these "new" ones can be skipped.